src/lib/helpers.ts

```diff
@@ -70,27 +74,33 @@ export const rpcMaxRequestsPerSecond = (chainId: number): number => {
   const envVarName = `RPC_REQUEST_RATE_LIMIT_${chainId}`;
   const envVarValue = process.env[envVarName];
 
-  // no rate limit provided in env var
-  if (!envVarValue) {
+  try {
+    return parseRpcMaxRequestsPerSecond(envVarValue);
+  } catch (e: any) {
+    throw new Error(`Environment variable parse error: '${envVarName}' was ${e.message}`);
+  }
+};
+
+export const parseRpcMaxRequestsPerSecond = (rawValue?: string): number => {
```

Comment:
@lightwalker-eth here's the split of functionality creating RPC config values. One is to read raw value from env vars, the other is to have it parsed into expected value range. We unit test the later.